src/core/actions/errors.js

```diff
@@ -0,0 +1,15 @@
+import { SET_ERROR } from 'core/constants';
+
+export function setError({ error, id } = {}) {
```

Comment:
Technically it's not needed, yes, but without it the error would be ``'_ref is undefined'`` which is way more confusing than ``'id cannot be empty'``. This is one thing I hate about ES6! Maybe the native error for this will be more sensical.